README.md

```diff
@@ -147,49 +147,34 @@ Rev 2 PCBs come with ARM 10-pin SWD headers and can be programmed via ST-Link. A
 ### ST-Link Software Installation
 
 ### Ubuntu
-1.  Download and build the ST-Link utilities:
-    ```
-    git clone https://github.com/stlink-org/stlink
-    cd stlink
-    cmake .
-    make
-    ```
-
-2. Copy the built binaries into their place:
-    ```
-    cd bin
-    sudo cp st-* /usr/local/bin
-    cd ../lib
-    sudo cp *.so* /lib32
-    ```
 
-3. Copy the udev rules to their place (this lets you run the st-link commands without using `sudo`):
-    ```
-    cd ../config/udev/rules.d
-    sudo cp 49-stlinkv* /etc/udev/rules.d/
-    ```
+1. sudo apt-install stlink-tools
+2. sudo apt-install stlink-gui
+3. sudo apt-install openocd
```

Comment:
should it be `sudo apt-get install`?

Reply:
Also I get a package not found error when I tried to install the first 2

Reply:
should be `sudo apt install` the `-get` has been unneeded for a long time

Reply:
thats weird that u cant find the first 2 packages. do a apt update maybe? @wmmc88 did the cmds work for u?

Reply:
Didnt try lol